packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -79,7 +80,9 @@ class Product extends Component {
         return {
             minHeight: imageHeight, // min-height instead of height so image will always align with grid bottom
             width: imageWidth,
-            backgroundImage: `url(${makeProductMediaPath(image.file)})`
+            backgroundImage: `url(${new ResourceUrl(
+                makeProductMediaPath(image.file)
+            ).toString()})`
```

Comment:
We could omit `.toString()` in places like this, right? It should be automatically called for expressions in template literal placeholders.

Reply:
Naming that method `toString` is the magic part, though. 😉